src/zino/api/legacy.py

```diff
@@ -249,6 +249,22 @@ async def do_getlog(self, case_id: Union[str, int]):
 
         self._respond_raw(".")
 
+    @requires_authentication
+    async def do_addhist(self, case_id: Union[str, int]):
+        try:
+            case_id = int(case_id)
+            event = self._state.events[case_id]
+        except (ValueError, KeyError):
+            return self._respond_error(f'event "{case_id}" does not exist')
+
+        self._respond(302, "please provide new history entry, terminate with '.'")
+        data = await self._read_multiline()
+        message = f"{self.user}\n" + "\n".join(line.strip() for line in data)
+        event.add_history(message)
```

Comment:
An example of the correct input (data) and what the message looks like (in a docstring) would be good here, more monday morning friendly.

Reply:
done